<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>SpendTime-v0.4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="6">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>行动表</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>时间记录表</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit">
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="displayFormat">
             <string>yyyy/MM/dd</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="6">
         <widget class="QTableView" name="todoTableView"/>
        </item>
        <item row="3" column="0" rowspan="4">
         <widget class="QTableView" name="timePlanTableView"/>
        </item>
        <item row="5" column="6">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>待办事项</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="2" rowspan="4">
         <widget class="QTableView" name="timeRecordTableView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QTableView" name="skillTableView"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>时间规划表</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="weekComboBox">
            <item>
             <property name="text">
              <string>星期一</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>星期二</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>星期三</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>星期四</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>星期五</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>星期六</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>星期日</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>输入框</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="skillLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>行动表</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skillAddButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>新增行动</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="skillDelButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>删除行动</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>待办事项</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="todoAddButton">
          <property name="text">
           <string>新增代办</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="todoDelButton">
          <property name="text">
           <string>删除代办</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="todoChangeButton">
          <property name="text">
           <string>切换状态</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>时间规划表</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addTimePlanButton">
          <property name="text">
           <string>添加时间规划</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delTimePlanButton">
          <property name="text">
           <string>删除时间规划</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setMusicButton">
          <property name="text">
           <string>设置提醒铃声</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>当前时间</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeLabel">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <fontweight>Thin</fontweight>
           </font>
          </property>
          <property name="text">
           <string>00:00:00</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>0</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="timerLineEdit"/>
      </item>
      <item>
       <widget class="ComplexLineEdit" name="shotTimeLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>25</string>
        </property>
        <property name="postfix" stdset="0">
         <string>m</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="timerButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>23</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>开始计时</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComplexLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">complexlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
